packages/contracts/package.json

```diff
@@ -15,7 +15,7 @@
     "lint": "forge fmt && eslint --fix --ignore-path .gitignore && yarn solhint './*(test|src)/**/*.sol'",
     "db-check": "tsc && node ./script/utils/checkDb.js",
     "db-export": "tsc && node ./script/utils/exporter.js",
-    "deploy": "make deploy-local",
+    "deploy": "make deploy-local && node script/generateTsAbis.js",
     "deploy:ci": "yarn db-export && make deploy-baseSepolia",
```

Comment:
better to create a yarn generateAbis and have deploy call that.  may want to consider doing it in the makefile as well.